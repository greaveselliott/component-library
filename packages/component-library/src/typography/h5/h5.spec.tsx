import React from 'react';
import { shallow } from 'enzyme';

import H5 from './h5.component';

test('Heading one should render correctly', () =>
  expect(shallow(<H5>abc</H5>)).toMatchSnapshot());
