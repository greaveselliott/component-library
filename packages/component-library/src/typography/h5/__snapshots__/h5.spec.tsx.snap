// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading one should render correctly 1`] = `
<h5
  className="heading-five"
>
  abc
</h5>
`;
